D/DynarePreprocessor/build_tarballs.jl

```diff
@@ -67,4 +73,4 @@ dependencies = [
     HostBuildDependency("flex_jll"),
 ]
 
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version=v"9")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version=v"13")
```

Comment:
Apart from the fact there is no version 13 of GCC, for maximum compatibility we want to use the _oldest_ version possible, not the latest one which would even result in binaries not compatible with any version of Julia.

Reply:
OK, I will search for the first gcc version to support  std::filesystem::path

Reply:
SDK 10.15 doesn't support syntax like
```
Environment.cc:31:17: error: no member named 'contains' in `'std::map<std::basic_string<char>,std::tuple<std::shared_ptr<macro::Function>,std::shared_ptr<macro::Expression>>>'
if (functions.contains(name))
Environment.cc:40:17: error: no member named 'contains' in 'std::map<std::basic_string<char>,std::shared_ptr<macro::Expression>>'
if (variables.contains(name))
SDK 11.3 doesn't complain about this syntax, but raises on error on exception constructs elsewhere in the original source code. 
I will search what is the best combination of minimal SDK version and patches.